### PR TITLE
Template Element

### PR DESCRIPTION
**Pull request by HakaCode**

### CODE REVIEW

**Comment by nazarepiedady** [comment]
@Peruibeloko, could you review this translated content to check if it is okay?

File: 8-web-components/4-template-element/article.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
 
-# Template element
+# Elemento de template
 
-A built-in `<template>` element serves as a storage for HTML markup templates. The browser ignores its contents, only checks for syntax validity, but we can access and use it in JavaScript, to create other elements.
+O elemento embutido `<template>` atua como um armazenamento para modelos de marcação HTML. O navegador ignora seu conteúdo, apenas verifica a validade da sintaxe, mas podemos acessá-lo e usá-lo em JavaScript para criar outros elementos.
```
Comment (Peruibeloko) [inline 8-web-components/4-template-element/article.md:4]:
```suggestion
O elemento nativo `<template>`
```

**Review by Peruibeloko — COMMENTED** [review]
Approved suggesting a minor change

**Comment by nazarepiedady** [comment]
@Peruibeloko, if possible, could you update or edit this pull request in order to be approved and merged?

**Review by jonnathan-ls — APPROVED** [review]
LGTM

I applied the changes (https://github.com/javascript-tutorial/pt.javascript.info/pull/417/commits/7981764b05e5229516eb78442d17ee867c7474ad) according to suggestions from @nazarepiedady and @Peruibeloko

**Review by nazarepiedady — APPROVED** [review]
It is fine now, so I am merging it to the main branch.

**Pull request merged**